<!DOCTYPE html>

<html lang="en">
    <head>
        
        <link rel="stylesheet" type="text/css" href="templates/slate.css">
            <meta http-equiv="CONTENT-TYPE" content="text/html; charset=UTF-8">
                <meta http-equiv="X-UA-Compatible" content="IE=edge">
                    <meta name="viewport" content="initial-scale=1, width=device-width">
                        <meta content="4DNucleome Hackathon" name="description">
                            <meta content="Projects" name="author">
                                
                                <!-- Global site tag (gtag.js) - Google Analytics -->
                                <script async src="https://www.googletagmanager.com/gtag/js?id=UA-169392828-1"></script>
                                <script>
                                    window.dataLayer = window.dataLayer || [];
                                    function gtag(){dataLayer.push(arguments);}
                                    gtag('js', new Date());
                                    gtag('config', 'UA-169392828-1');
                                </script>
                                    
                                <title>4D Nucleome Hackathon</title>
                                <style>
                                    a:unvisited {color: lightseagreen;}
                                    a:visited {color: lightseagreen;}
                                    a:link {color: lightseagreen;}
                                    a:hover {color: white;}
                                    b {color: lightseagreen;}
                                    body {font-family: 'Book Antiqua', Palatino, serif';font-size: 25px;}
                                </style>
                                <div class="jumbotron">
                                    <img src="4dn_logo_raster.png" style="border-radius: 50%; height: 100px; margin-right: 20px;">
                                    <h1 style="font-size:50px">4D Nucleome Hackathon</h1>
                                    <hr>
                                </div>
                                <div class="jumbotron">
                                <p style="font-size:18px", align="justify">
                                <a href="#Project 1">Project 1</a> &nbsp;&bull;&nbsp
                                <a href="#Project 2">Project 2</a> &nbsp;&bull;&nbsp
                                <a href="#Project 3">Project 3</a> &nbsp;&bull;&nbsp
                                <a href="#Project 4">Project 4</a> &nbsp;&bull;&nbsp
                                <a href="#Project 5">Project 5</a>
                                </p>
                                </div>
                                

    
    </head>
                                        
    <body role="document" style='font="Book Antiqua', Palatino, serif"'>
        
        <a name="Project 1" id="Project 1"><div class="jumbotron" name="Project 1" id="Project 1"><a></a>
                <div class="page-header">
                <h2 style="font-size:26px"><b>Project 1</b></h2>
                <!--<img align="left" src="4dn_logo_raster.png" style="display:inline-block; padding-right:10px; height: 100px; margin-right: 20px;">-->
                <!--<img align="left" src="4dn_logo_raster.png" style="border-radius: 50%; height: 100px; margin-right: 20px;">-->
                <img align="left" src="4dn_logo_raster.png" style="float:left; padding-right:10px; height: 100px; margin-right: 20px;">
                <span style="position:absolute; display: inline; bottom: 10px; margin-right: 40px;">
                    <p style="font-size:20px", align="justify">
                        <b>Leading lab</b>: Span 21 <br>
                        <b>Stakeholders</b>: Anupama Jha (anupamaj [ a t ] uw.edu) and Gang Li (gangliuw [ a t ] uw.edu) <br>
                        <b>Hackathon project title</b>: Integrating single-cell Hi-C and  single-cell RNA sequencing data <br>
                        <b>Desired deliverable</b>:  Benchmarking results comparing multiple  existing software tools for integrating scHiC and scRNA data. <br>
                        <b>Expected coding experience level</b>: Intermediate-Advanced. <br>
                        <b>Abstract</b>: Recently, Liu et al. developed HiRES, a  multi-omics sequencing approach to simultaneously profile 3D chromatin contacts  and gene expression in single cells. The HiRES assay thus directly links  chromatin conformation and gene expression profiles within single cells.  Previous studies that aimed to investigate the interplay between chromatin  interactions and transcriptomic profiles necessarily generated unpaired  single-cell RNA-sequencing and chromatin conformation data. For the hackathon,  we will use the newly available HiRES data to benchmark existing software tools  developed for matching cells across modalities. In particular, we evaluate  GLUE, LS-MMDMA (<a href="https://academic.oup.com/bioinformatics/article/39/7/btad420/7221538" target="_blank" class="aaastandout">Meng 2023</a>), Pomona (<a href="https://pubmed.ncbi.nlm.nih.gov/34398192/" target="_blank" class="aaastandout">Cao  2021</a>), SCOT (<a href="https://pubmed.ncbi.nlm.nih.gov/35050714/" target="_blank" class="aaastandout">Demetci 2022</a>), Synmatch (<a href="https://academic.oup.com/bioinformatics/article/38/Supplement_2/ii148/6702005?login=false" target="_blank" class="aaastandout">Hristov 2022</a>), and CMOT (<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02989-8" target="_blank" class="aaastandout">Alatkar 2023</a>). In addition, we will consider  four different techniques for representing the scHi-C contact matrices,  including the contact decay profile, the HiCRep (<a href="https://academic.oup.com/bioinformatics/article/37/18/2996/6133255?login=false" target="_blank" class="aaastandout">Lin 2021</a>) similarity score, a latent Dirichlet  allocation model (<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008173" target="_blank" class="aaastandout">Kim 2020</a>), and the scGAD method (<a href="https://academic.oup.com/bioinformatics/article/38/14/3642/6598798" target="_blank" class="aaastandout">Shen 2022</a>). Our results will compare and  contrast the performance of these tools and Hi-C representation techniques in  the context of integration with scRNA-seq data. For the hackathon, we will use co-assay data  to validate integration of scRNA-seq and scHi-C. <br>
                    </p>
                </span>
                
                
        </div></div></a>

    </body>

</html>